server/pulp/plugins/util/publish_step.py

```diff
@@ -364,7 +380,7 @@ class PluginStep(Step):
     """
 
     def __init__(self, step_type, repo=None, conduit=None, config=None, working_dir=None,
-                 plugin_type=None):
+                 plugin_type=None, **kwargs):
```

Comment:
There are some kwargs (such as disable_reporting) that the PluginStep don't care about but should be passed up the chain.